### PR TITLE
feat: Add tags for Glue Databases

### PR DESCRIPTION
**Pull request by erezrokah**
#### Summary

Brings https://github.com/cloudquery/cq-provider-aws/pull/1420 into the monorepo

---

Use the following steps to ensure your PR is ready to be reviewed

- [x] Read the [contribution guidelines](../blob/main/CONTRIBUTING.md) 🧑‍🎓
- [x] Test locally on your own infrastructure
- [x] Run `go fmt` to format your code 🖊
- [x] Lint your changes via `golangci-lint run` 🚨 (install golangci-lint [here](https://golangci-lint.run/usage/install/#local-installation)) 
- [x] Update or add tests 🧪
- [x] Ensure the status checks below are successful ✅

### CODE REVIEW

**Review by bbernays — CHANGES_REQUESTED** [review]
I think we also need to update the `resources/services/glue/databases.hcl` file

**Comment by erezrokah** [comment]
> I think we also need to update the `resources/services/glue/databases.hcl` file

Thanks @bbernays. https://github.com/cloudquery/cloudquery/pull/1326/commits/22c696bc91ccc206b7cef7ba468413c990dfd61e should handle and it and ensure other `hcl` files are in sync too

File: plugins/source/aws/resources/services/glue/databases.hcl [hunk 1]
```diff
@@ -40,6 +41,10 @@ resource "aws" "glue" "databases" {
     description       = "The Amazon Resource Name (ARN) of the workflow."
     generate_resolver = true
   }
+  userDefinedColumn "tags" {
+    type              = "json"
```
Comment (bbernays) [inline plugins/source/aws/resources/services/glue/databases.hcl:45]: Do we need a description?

**Review by bbernays — APPROVED** [review]
LGTM... only single nit

File: plugins/source/aws/resources/services/glue/databases.hcl (hunk 1 above)
Comment (erezrokah) [reply plugins/source/aws/resources/services/glue/databases.hcl:45]: Done in https://github.com/cloudquery/cloudquery/pull/1326/commits/27abbf9fa677e939adac71327e40e5dda8c99957!

**Pull request merged**